src/core/parser/parse_uri.c

```diff
@@ -1439,3 +1439,26 @@ void proto_type_to_str(unsigned short type, str *s) {
 		*s = s_null;
 	}
 }
+
+void proto_type_int_to_str(int type, str *s) {
```

Comment:
What about just using the existing function proto_type_to_str? You can easily cast the input parameter from int to unsigned short, or I am wrong?

Reply:
i got a error for conversion .Now fixed it in next commit